components/director/internal/authnmappinghandler/handler_test.go

```diff
@@ -574,7 +574,7 @@ func TestHandler(t *testing.T) {
 		handler.ServeHTTP(w, req)
 
 		resp := w.Result()
-		require.Equal(t, http.StatusOK, resp.StatusCode)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
```

Comment:
wouldnt 401 be better ?

Reply:
https://github.com/ory/oathkeeper/blob/48603a1ac484b6571706021f2667f770604256b6/pipeline/mutate/mutator_hydrator.go#L218

Reply:
I don't mind really - I was wondering about that. I'll change it up, sure